### PR TITLE
Nate/feature/LOOP-1293/cgm status report

### PR DESCRIPTION
**Pull request by nhamming**
https://tidepool.atlassian.net/browse/LOOP-1293

### CODE REVIEW

File: LoopUI/Views/CGMStatusHUDView.swift [hunk 1]
```diff
@@ -83,96 +101,27 @@ public final class CGMStatusHUDView: DeviceStatusHUDView, NibLoadable {
...
         
-        let time = timeFormatter.string(from: glucoseStartDate)
-        caption?.text = time
+        glucoseValueHUD.glucoseLabel.text = viewModel.glucoseValueString
+        glucoseValueHUD.unitLabel.text = viewModel.unitsString
+        glucoseTrendHUD.setTrend(viewModel.trend)
+        tintColor = viewModel.tintColor ?? .label
+        accessibilityValue = viewModel.accessibilityString
         
-        isStaleAt = glucoseStartDate.addingTimeInterval(staleGlucoseAge)
```
Comment (ps2) [inline LoopUI/Views/CGMStatusHUDView.swift:94]: If the CGM goes offline; i.e. phone is out of range of the sensor, and the CGMManager does not report any data, we still need to show the user that the data is stale after a period of time. It appears that that behavior is being removed here.

**Review by ps2 — CHANGES_REQUESTED** [review]

File: LoopUI/Views/CGMStatusHUDView.swift (hunk 1 above)
Comment (nhamming) [reply LoopUI/Views/CGMStatusHUDView.swift:94]: @ps2 This code was moved to the `CGMStatusHUDViewModel` as suggested by @rickpasetto. There is a timer that when fired would execute the block passed from the `CGMStatuHUDView`. Does this answer your question?

Comment (ps2) [reply LoopUI/Views/CGMStatusHUDView.swift:94]: Ah, ok. Thanks. Yes, that answers it. Will approve.

**Review by ps2 — APPROVED** [review]
LGTM!

**Pull request merged**